install/kubernetes/helm/istio/requirements.yaml

```diff
@@ -1,12 +1,13 @@
 dependencies:
   - name: pilot
-    version: 0.1.0
-    # condition: global.pilot.enabled
+    version: 0.3.0
+    condition: pilot.enabled
   - name: mixer
-    version: 0.1.0
-    # condition: global.mixer.enabled
+    version: 0.3.0
+    condition: mixer.enabled
   - name: initializer
-    version: 0.1.0
-    # condition: global.initializer.enabled
+    version: 0.3.0
+    condition: initializer.enabled
   - name: security
-    version: 0.1.0
+    version: 0.3.0
+    condition: gloabl.securityEnabled
```

Comment:
gloabl is a typo

also, why not global.security.enabled?

Reply:
I'm not sure how the template rendered properly with the typo or why the helm linter didn't catch this.  Perhaps security wasn't enabled in the test case you ran?

Reply:
to don't need to check each level since there are no more values for `security`
https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/values.md#flat-or-nested-values

Reply:
it just didn't find that variable so this `{{- if .Values.global.securityEnabled }}` parse to false but it doesn't fail

Reply:
helm chart best practices make sense there - sort of like a bullet point.  In the future, I suspect more config options will end up in security at which point we can break it into a new level.  Quick Q - is the camelcase a best practice as well?

Cheers
-steve

Reply:
yes, camelcase is standar also